common/src/test/scala/org/broadinstitute/monster/etl/MsgTransformationsSpec.scala

```diff
@@ -60,6 +60,84 @@ class MsgTransformationsSpec extends FlatSpec with Matchers {
     )
   }
 
+  // removeFields
+  it should "remove fields in object messages" in {
+    val input = Obj(
+      Str("foo") -> Str("bar"),
+      Str("foobar") -> Int32(123),
+      Str("baz") -> Arr(Str("qux"), Float64(1.23))
+    )
+
+    val removed =
+      MsgTransformations.removeFields(Set("foo"))(input)
```

Comment:
Can you test with two fields? Just to be sure the looping is actually happening.